### PR TITLE
[NumberField] Add sbyte type

### PR DESCRIPTION
**Pull request by franklupo**
Add sbyte type to FluentNumberField

### CODE REVIEW

File: src/Core/Components/Base/InputHelpers.cs [hunk 1]
```diff
@@ -51,17 +50,27 @@ public static string GetMinValue()
         return value;
     }
 
-    internal static void ValidateIntegerInputs(string? max, string? min)
+
```
Comment (dvoituron) [inline src/Core/Components/Base/InputHelpers.cs:53]: Can you Put only 1 space line?

File: src/Core/Components/Base/InputHelpers.cs [hunk 2]
```diff
@@ -51,17 +50,27 @@ public static string GetMinValue()
...
+    internal static void ValidateSByteInputs(string max, string min)
     {
+        var maxValue = Convert.ToSByte(max, CultureInfo.InvariantCulture);
+        var minValue = Convert.ToSByte(min, CultureInfo.InvariantCulture);
+
+        if (maxValue < minValue)
+        {
+            throw new ArgumentException("Float Max value is smaller than Min value.");
```
Comment (dvoituron) [inline src/Core/Components/Base/InputHelpers.cs:61]: Message incorrect

File: src/Core/Components/Base/InputHelpers.cs [hunk 3]
```diff
@@ -51,17 +50,27 @@ public static string GetMinValue()
         return value;
     }
 
-    internal static void ValidateIntegerInputs(string? max, string? min)
+
+    internal static void ValidateSByteInputs(string max, string min)
```
Comment (dvoituron) [inline src/Core/Components/Base/InputHelpers.cs:54]: Why arguments are nullable in all other methods and not your new one?

**Review by dvoituron — CHANGES_REQUESTED** [review]

File: src/Core/Components/Base/InputHelpers.cs (hunk 3 above)
Comment (franklupo) [reply src/Core/Components/Base/InputHelpers.cs:54]: `Convert.ToSByte ` not accept null value

Comment (vnbaaij) [reply src/Core/Components/Base/InputHelpers.cs:53]: Docs say yes:
image

**Review by dvoituron — APPROVED** [review]

**Pull request merged**